homework-19/script.js

```diff
@@ -0,0 +1,82 @@
+class ApartamentBuilding {
+    constructor(street, number, floorsNumber, apartamentsNumber, dwellersNumber) {
+        this.adress = {sreet: street, number: number};
+        this.floorsNumber = floorsNumber;
+        this.apartamentsNumber = apartamentsNumber;
+        this.apartamentsArray = [0];
+        this.dwellersNumber = dwellersNumber;
+    }
+
+    addApartamentInfo(apartament) {
```

Comment:
```suggestion
    addApartament(apartament) {
```

Ви додаєте екземпляр класу Квартира, а не інформацію про неї